raqm.c

```diff
@@ -649,14 +656,14 @@ raqm_shape_u32 (unsigned int* u32_str,
     {
         hb_glyph_info = hb_buffer_get_glyph_infos (runs[i].hb_buffer, &glyph_count);
         hb_glyph_position = hb_buffer_get_glyph_positions (runs[i].hb_buffer, &postion_length);
-        int j;
-        for (j = 0; j < glyph_count; j++)
+
+        for (k = 0; k < glyph_count; k++)
```

Comment:
I see, I’d just move `j` declaration 2 lines above and make it `unsigned` as well.